packages/swift-sdk/build_ios.sh

```
@@ -46,6 +48,31 @@ CLEAN=false
+
+  mkdir -p "$target_staging_dir"
+  cp "$library" "$target_staging_dir/"
+  cp -R "$headers" "$target_staging_dir/include"
+
+  STAGED_LIB="$target_staging_dir/$(basename "$library")"
+  STAGED_HEADERS="$target_staging_dir/include"
+
+  # The final static library and generated headers are all xcodebuild needs.
+  # Release the much larger per-architecture dependency tree before building
+  # the next target so persistent CI runners cannot exhaust their disk.
+  rm -rf "$TARGET_DIR/$target"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)